comps/retrievers/deployment/docker_compose/compose.yaml

```diff
@@ -170,6 +171,26 @@ services:
         condition: service_healthy
       tei-embedding-serving:
         condition: service_healthy
+  
+  retriever-arango:
+    extends: retriever
+    container_name: retriever-arango
+    depends_on:
+      - arango-vector-db
+      - tei-embedding-service
+    environment:
+      no_proxy: ${no_proxy}
```

Comment:
we're missing the `RETRIEVER_COMPONENT_NAME` environment (see `retriever-vdms` above for an example)